content/webapp/components/ArchiveTree/index.tsx

```diff
@@ -206,7 +203,10 @@ const ArchiveTree: FunctionComponent<{ work: Work }> = ({
     }
   }, [archiveTree, tabbableId]);
 
-  const selected = useRef<HTMLAnchorElement>(null);
+  useEffect(() => {
+    // On mobile we want to close the archive tree if a user selects a work
+    setShowArchiveTree(false);
```

Comment:
How does this only target mobile? Or do we display the archive tree on desktop even when `showArchiveTree` is `false`?

Reply:
The archive tree is displayed in a Modal on small screens and this is the value that gets passed in to the Modal's isActive property - I should probably rename it to something more descriptive